commander_unix.yaml

```diff
@@ -126,6 +126,7 @@ tests:
     exit-code: 0
 
   it should execute test from url:
+    skip: true #TODO: circle ci does not resolve the URL properly
```

Comment:
I may not have found the right pipeline failure (I’m only on a phone at the moment), but I can only find where the name resolution for the image pull is failing. Did you mean to remove this?

Reply:
I saw this at least sometime to be flaky because of DNS issues, can't show you a build though I think :/

Reply:
Yeah, I saw it in my testing too..